src/marqo/tensor_search/models/add_docs_objects.py

```diff
@@ -30,7 +30,7 @@ class Config:
     modelAuth: Optional[ModelAuth] = None
     mappings: Optional[dict] = None
     documents: Union[Sequence[Union[dict, Any]], np.ndarray]
-    imageDownloadThreadCount: int = 20
+    imageDownloadThreadCount: int = read_env_vars_and_defaults_ints(EnvVars.MARQO_MEDIA_DOWNLOAD_THREAD_COUNT)
```

Comment:
I don't think we can do this since index_management deals with index settings right? And this one is an add docs object.

but I did push new code which:

- adds a new env var. Now we have two env vars for download threads: (1) image threadcount, and (2) media thread count. This is for depreciating the image one. Both can't be set explicitly at the same time or it will error.
- in tensor_search, a new method which selects the proper threadcount variable to use andsets the default media for languagebind to 5 threads.

Reply:
got it